dyn_em/module_diffusion_em.F

```diff
@@ -5160,7 +5160,7 @@ SUBROUTINE nonlocal_flux (config_flags,nlflux,gamu,gamv,hpbl,kpbl,     &
   DO j = j_start,j_end
   DO i = i_start,i_end
      IF(pblflg(i,j))THEN
-       k = kpbl(i,j) - 1
+       k = max(kpbl(i,j) - 1,1)
```

Comment:
pblflg instead should be set false if kpbl.le.1 (see YSU)

Reply:
Jimy,
Yes, I see the error in logic in the following IF test for pblflg:
```
   5121   DO j = j_start, j_end
   5122   DO i = i_start, i_end
   5123      IF(hpbl(i,j).LT.zq(i,2,j)) kpbl(i,j) = 1
   5124      IF(kpbl(i,j).LT.1) pblflg(i,j) = .false.
   5125   ENDDO
   5126   ENDDO
```
Line 5124 should be
```
   5124      IF(kpbl(i,j).LE.1) pblflg(i,j) = .false.
```

Reply:
@dudhia 
Jimy,
OK, this is now done as you suggested.